modules/apps/bean-portlet/bean-portlet-cdi-extension-sample-test/src/main/webapp/WEB-INF/liferay-portlet.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!DOCTYPE liferay-portlet-app PUBLIC "-//Liferay//DTD Portlet Application 7.4.0//EN" "http://www.liferay.com/dtd/liferay-portlet-app_7_4_0.dtd">
+
+<liferay-portlet-app>
+	<portlet>
+		<portlet-name>1</portlet-name>
```

Comment:
@Kvale1733 Can we use a more meaningful name?

Reply:
I'll change the name